internal/service/ecs/service.go

```diff
@@ -1441,12 +1471,49 @@ func expandServices(srv []interface{}) []*ecs.ServiceConnectService {
 			config.PortName = aws.String(v)
 		}
 
+		if v, ok := raw["tls"].([]interface{}); ok && len(v) > 0 {
```

Comment:
Please run all acceptance tests to verify that a nil/non-provided `tls` value does not panic

Reply:
I have run all the tests and they all pass except: TestAccECSTaskDefinition_fsxWinFileSystem. This seems unrelated to this change and not sure why the provisioning of the FSX Windows File System fails:

==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./internal/service/ecs/... -v -count 1 -parallel 10   -timeout 360m
=== RUN   TestValidPlacementConstraint
=== PAUSE TestValidPlacementConstraint
=== RUN   TestValidPlacementStrategy
=== PAUSE TestValidPlacementStrategy
=== RUN   TestAccECSAccountSettingDefault_containerInstanceLongARNFormat
=== PAUSE TestAccECSAccountSettingDefault_containerInstanceLongARNFormat
=== RUN   TestAccECSAccountSettingDefault_serviceLongARNFormat
=== PAUSE TestAccECSAccountSettingDefault_serviceLongARNFormat
=== RUN   TestAccECSAccountSettingDefault_taskLongARNFormat
=== PAUSE TestAccECSAccountSettingDefault_taskLongARNFormat
=== RUN   TestAccECSAccountSettingDefault_vpcTrunking
=== PAUSE TestAccECSAccountSettingDefault_vpcTrunking
=== RUN   TestAccECSAccountSettingDefault_containerInsights
=== PAUSE TestAccECSAccountSettingDefault_containerInsights
=== RUN   TestAccECSCapacityProvider_basic
=== PAUSE TestAccECSCapacityProvider_basic
=== RUN   TestAccECSCapacityProvider_disappears
=== PAUSE TestAccECSCapacityProvider_disappears
=== RUN   TestAccECSCapacityProvider_managedScaling
=== PAUSE TestAccECSCapacityProvider_managedScaling
=== RUN   TestAccECSCapacityProvider_managedScalingPartial
=== PAUSE TestAccECSCapacityProvider_managedScalingPartial
=== RUN   TestAccECSCapacityProvider_tags
=== PAUSE TestAccECSCapacityProvider_tags
=== RUN   TestAccECSClusterCapacityProviders_basic
=== PAUSE TestAccECSClusterCapacityProviders_basic
=== RUN   TestAccECSClusterCapacityProviders_disappears
=== PAUSE TestAccECSClusterCapacityProviders_disappears
=== RUN   TestAccECSClusterCapacityProviders_defaults
=== PAUSE TestAccECSClusterCapacityProviders_defaults
=== RUN   TestAccECSClusterCapacityProviders_destroy
=== PAUSE TestAccECSClusterCapacityProviders_destroy
=== RUN   TestAccECSClusterCapacityProviders_Update_capacityProviders
=== PAUSE TestAccECSClusterCapacityProviders_Update_capacityProviders
=== RUN   TestAccECSClusterCapacityProviders_Update_defaultStrategy
=== PAUSE TestAccECSClusterCapacityProviders_Update_defaultStrategy
=== RUN   TestAccECSClusterDataSource_ecsCluster
=== PAUSE TestAccECSClusterDataSource_ecsCluster
=== RUN   TestAccECSClusterDataSource_ecsClusterContainerInsights
=== PAUSE TestAccECSClusterDataSource_ecsClusterContainerInsights
=== RUN   TestAccECSClusterDataSource_tags
=== PAUSE TestAccECSClusterDataSource_tags
=== RUN   TestAccECSCluster_basic
=== PAUSE TestAccECSCluster_basic
=== RUN   TestAccECSCluster_disappears
=== PAUSE TestAccECSCluster_disappears
=== RUN   TestAccECSCluster_tags
=== PAUSE TestAccECSCluster_tags
=== RUN   TestAccECSCluster_serviceConnectDefaults
=== PAUSE TestAccECSCluster_serviceConnectDefaults
=== RUN   TestAccECSCluster_containerInsights
=== PAUSE TestAccECSCluster_containerInsights
=== RUN   TestAccECSCluster_configuration
=== PAUSE TestAccECSCluster_configuration
=== RUN   TestAccECSContainerDefinitionDataSource_ecsContainerDefinition
=== PAUSE TestAccECSContainerDefinitionDataSource_ecsContainerDefinition
=== RUN   TestAccECSServiceDataSource_basic
=== PAUSE TestAccECSServiceDataSource_basic
=== RUN   TestEndpointConfiguration
=== RUN   TestEndpointConfiguration/service_config_file
=== RUN   TestEndpointConfiguration/base_endpoint_config_file
=== RUN   TestEndpointConfiguration/package_name_endpoint_config_overrides_aws_service_envvar
=== RUN   TestEndpointConfiguration/base_endpoint_envvar
=== RUN   TestEndpointConfiguration/package_name_endpoint_config
=== RUN   TestEndpointConfiguration/package_name_endpoint_config_overrides_base_config_file
=== RUN   TestEndpointConfiguration/service_aws_envvar
=== RUN   TestEndpointConfiguration/service_aws_envvar_overrides_base_config_file
=== RUN   TestEndpointConfiguration/base_endpoint_envvar_overrides_base_config_file
=== RUN   TestEndpointConfiguration/package_name_endpoint_config_overrides_base_envvar
=== RUN   TestEndpointConfiguration/package_name_endpoint_config_overrides_service_config_file
=== RUN   TestEndpointConfiguration/service_aws_envvar_overrides_base_envvar
=== RUN   TestEndpointConfiguration/service_aws_envvar_overrides_service_config_file
=== RUN   TestEndpointConfiguration/base_endpoint_envvar_overrides_service_config_file
=== RUN   TestEndpointConfiguration/service_config_file_overrides_base_config_file
=== RUN   TestEndpointConfiguration/no_config
--- PASS: TestEndpointConfiguration (0.64s)
    --- PASS: TestEndpointConfiguration/service_config_file (0.04s)
    --- PASS: TestEndpointConfiguration/base_endpoint_config_file (0.03s)
    --- PASS: TestEndpointConfiguration/package_name_endpoint_config_overrides_aws_service_envvar (0.05s)
    --- PASS: TestEndpointConfiguration/base_endpoint_envvar (0.03s)
    --- PASS: TestEndpointConfiguration/package_name_endpoint_config (0.04s)
    --- PASS: TestEndpointConfiguration/package_name_endpoint_config_overrides_base_config_file (0.09s)
    --- PASS: TestEndpointConfiguration/service_aws_envvar (0.08s)
    --- PASS: TestEndpointConfiguration/service_aws_envvar_overrides_base_config_file (0.03s)
    --- PASS: TestEndpointConfiguration/base_endpoint_envvar_overrides_base_config_file (0.03s)
    --- PASS: TestEndpointConfiguration/package_name_endpoint_config_overrides_base_envvar (0.04s)
    --- PASS: TestEndpointConfiguration/package_name_endpoint_config_overrides_service_config_file (0.04s)
    --- PASS: TestEndpointConfiguration/service_aws_envvar_overrides_base_envvar (0.02s)
    --- PASS: TestEndpointConfiguration/service_aws_envvar_overrides_service_config_file (0.03s)
    --- PASS: TestEndpointConfiguration/base_endpoint_envvar_overrides_service_config_file (0.02s)
    --- PASS: TestEndpointConfiguration/service_config_file_overrides_base_config_file (0.02s)
    --- PASS: TestEndpointConfiguration/no_config (0.02s)
=== RUN   Test_GetRoleNameFromARN
=== PAUSE Test_GetRoleNameFromARN
=== RUN   Test_GetClustereNameFromARN
=== PAUSE Test_GetClustereNameFromARN
=== RUN   TestAccECSService_basic
=== PAUSE TestAccECSService_basic
=== RUN   TestAccECSService_basicImport
=== PAUSE TestAccECSService_basicImport
=== RUN   TestAccECSService_disappears
=== PAUSE TestAccECSService_disappears
=== RUN   TestAccECSService_PlacementStrategy_unnormalized
=== PAUSE TestAccECSService_PlacementStrategy_unnormalized
=== RUN   TestAccECSService_CapacityProviderStrategy_basic
=== PAUSE TestAccECSService_CapacityProviderStrategy_basic
=== RUN   TestAccECSService_CapacityProviderStrategy_forceNewDeployment
=== PAUSE TestAccECSService_CapacityProviderStrategy_forceNewDeployment
=== RUN   TestAccECSService_CapacityProviderStrategy_update
=== PAUSE TestAccECSService_CapacityProviderStrategy_update
=== RUN   TestAccECSService_familyAndRevision
=== PAUSE TestAccECSService_familyAndRevision
=== RUN   TestAccECSService_healthCheckGracePeriodSeconds
=== PAUSE TestAccECSService_healthCheckGracePeriodSeconds
=== RUN   TestAccECSService_iamRole
=== PAUSE TestAccECSService_iamRole
=== RUN   TestAccECSService_DeploymentControllerType_codeDeploy
=== PAUSE TestAccECSService_DeploymentControllerType_codeDeploy
=== RUN   TestAccECSService_DeploymentControllerType_codeDeployUpdateDesiredCountAndHealthCheckGracePeriod
=== PAUSE TestAccECSService_DeploymentControllerType_codeDeployUpdateDesiredCountAndHealthCheckGracePeriod
=== RUN   TestAccECSService_DeploymentControllerType_external
=== PAUSE TestAccECSService_DeploymentControllerType_external
=== RUN   TestAccECSService_alarmsAdd
=== PAUSE TestAccECSService_alarmsAdd
=== RUN   TestAccECSService_alarmsUpdate
=== PAUSE TestAccECSService_alarmsUpdate
=== RUN   TestAccECSService_DeploymentValues_basic
=== PAUSE TestAccECSService_DeploymentValues_basic
=== RUN   TestAccECSService_DeploymentValues_minZeroMaxOneHundred
=== PAUSE TestAccECSService_DeploymentValues_minZeroMaxOneHundred
=== RUN   TestAccECSService_deploymentCircuitBreaker
=== PAUSE TestAccECSService_deploymentCircuitBreaker
=== RUN   TestAccECSService_loadBalancerChanges
=== PAUSE TestAccECSService_loadBalancerChanges
=== RUN   TestAccECSService_clusterName
=== PAUSE TestAccECSService_clusterName
=== RUN   TestAccECSService_alb
=== PAUSE TestAccECSService_alb
=== RUN   TestAccECSService_multipleTargetGroups
=== PAUSE TestAccECSService_multipleTargetGroups
=== RUN   TestAccECSService_forceNewDeployment
=== PAUSE TestAccECSService_forceNewDeployment
=== RUN   TestAccECSService_forceNewDeploymentTriggers
=== PAUSE TestAccECSService_forceNewDeploymentTriggers
=== RUN   TestAccECSService_PlacementStrategy_basic
=== PAUSE TestAccECSService_PlacementStrategy_basic
=== RUN   TestAccECSService_PlacementStrategy_missing
=== PAUSE TestAccECSService_PlacementStrategy_missing
=== RUN   TestAccECSService_PlacementConstraints_basic
=== PAUSE TestAccECSService_PlacementConstraints_basic
=== RUN   TestAccECSService_PlacementConstraints_emptyExpression
=== PAUSE TestAccECSService_PlacementConstraints_emptyExpression
=== RUN   TestAccECSService_LaunchTypeFargate_basic
=== PAUSE TestAccECSService_LaunchTypeFargate_basic
=== RUN   TestAccECSService_LaunchTypeFargate_platformVersion
=== PAUSE TestAccECSService_LaunchTypeFargate_platformVersion
=== RUN   TestAccECSService_LaunchTypeFargate_waitForSteadyState
=== PAUSE TestAccECSService_LaunchTypeFargate_waitForSteadyState
=== RUN   TestAccECSService_LaunchTypeFargate_updateWaitForSteadyState
=== PAUSE TestAccECSService_LaunchTypeFargate_updateWaitForSteadyState
=== RUN   TestAccECSService_LaunchTypeEC2_network
=== PAUSE TestAccECSService_LaunchTypeEC2_network
=== RUN   TestAccECSService_DaemonSchedulingStrategy_basic
=== PAUSE TestAccECSService_DaemonSchedulingStrategy_basic
=== RUN   TestAccECSService_DaemonSchedulingStrategy_setDeploymentMinimum
=== PAUSE TestAccECSService_DaemonSchedulingStrategy_setDeploymentMinimum
=== RUN   TestAccECSService_replicaSchedulingStrategy
=== PAUSE TestAccECSService_replicaSchedulingStrategy
=== RUN   TestAccECSService_ServiceRegistries_basic
=== PAUSE TestAccECSService_ServiceRegistries_basic
=== RUN   TestAccECSService_ServiceRegistries_container
=== PAUSE TestAccECSService_ServiceRegistries_container
=== RUN   TestAccECSService_ServiceRegistries_changes
=== PAUSE TestAccECSService_ServiceRegistries_changes
=== RUN   TestAccECSService_ServiceRegistries_removal
=== PAUSE TestAccECSService_ServiceRegistries_removal
=== RUN   TestAccECSService_ServiceConnect_basic
=== PAUSE TestAccECSService_ServiceConnect_basic
=== RUN   TestAccECSService_ServiceConnect_full
=== PAUSE TestAccECSService_ServiceConnect_full
=== RUN   TestAccECSService_ServiceConnect_ingressPortOverride
=== PAUSE TestAccECSService_ServiceConnect_ingressPortOverride
=== RUN   TestAccECSService_ServiceConnect_remove
=== PAUSE TestAccECSService_ServiceConnect_remove
=== RUN   TestAccECSService_Tags_basic
=== PAUSE TestAccECSService_Tags_basic
=== RUN   TestAccECSService_Tags_managed
=== PAUSE TestAccECSService_Tags_managed
=== RUN   TestAccECSService_Tags_propagate
=== PAUSE TestAccECSService_Tags_propagate
=== RUN   TestAccECSService_executeCommand
=== PAUSE TestAccECSService_executeCommand
=== RUN   TestAccECSTag_basic
=== PAUSE TestAccECSTag_basic
=== RUN   TestAccECSTag_disappears
=== PAUSE TestAccECSTag_disappears
=== RUN   TestAccECSTag_ResourceARN_batchComputeEnvironment
=== PAUSE TestAccECSTag_ResourceARN_batchComputeEnvironment
=== RUN   TestAccECSTag_value
=== PAUSE TestAccECSTag_value
=== RUN   TestAccECSTaskDefinitionDataSource_ecsTaskDefinition
=== PAUSE TestAccECSTaskDefinitionDataSource_ecsTaskDefinition
=== RUN   TestContainerDefinitionsAreEquivalent_basic
=== PAUSE TestContainerDefinitionsAreEquivalent_basic
=== RUN   TestContainerDefinitionsAreEquivalent_portMappings
=== PAUSE TestContainerDefinitionsAreEquivalent_portMappings
=== RUN   TestContainerDefinitionsAreEquivalent_portMappingsIgnoreHostPort
=== PAUSE TestContainerDefinitionsAreEquivalent_portMappingsIgnoreHostPort
=== RUN   TestContainerDefinitionsAreEquivalent_arrays
=== PAUSE TestContainerDefinitionsAreEquivalent_arrays
=== RUN   TestContainerDefinitionsAreEquivalent_negative
=== PAUSE TestContainerDefinitionsAreEquivalent_negative
=== RUN   TestContainerDefinitionsAreEquivalent_missingEnvironmentName
=== PAUSE TestContainerDefinitionsAreEquivalent_missingEnvironmentName
=== RUN   Test_StripRevision
=== PAUSE Test_StripRevision
=== RUN   TestAccECSTaskDefinition_basic
=== PAUSE TestAccECSTaskDefinition_basic
=== RUN   TestAccECSTaskDefinition_scratchVolume
=== PAUSE TestAccECSTaskDefinition_scratchVolume
=== RUN   TestAccECSTaskDefinition_DockerVolume_basic
=== PAUSE TestAccECSTaskDefinition_DockerVolume_basic
=== RUN   TestAccECSTaskDefinition_DockerVolume_minimal
=== PAUSE TestAccECSTaskDefinition_DockerVolume_minimal
=== RUN   TestAccECSTaskDefinition_runtimePlatform
=== PAUSE TestAccECSTaskDefinition_runtimePlatform
=== RUN   TestAccECSTaskDefinition_Fargate_runtimePlatform
=== PAUSE TestAccECSTaskDefinition_Fargate_runtimePlatform
=== RUN   TestAccECSTaskDefinition_Fargate_runtimePlatformWithoutArch
=== PAUSE TestAccECSTaskDefinition_Fargate_runtimePlatformWithoutArch
=== RUN   TestAccECSTaskDefinition_EFSVolume_minimal
=== PAUSE TestAccECSTaskDefinition_EFSVolume_minimal
=== RUN   TestAccECSTaskDefinition_EFSVolume_basic
=== PAUSE TestAccECSTaskDefinition_EFSVolume_basic
=== RUN   TestAccECSTaskDefinition_EFSVolume_transitEncryptionMinimal
=== PAUSE TestAccECSTaskDefinition_EFSVolume_transitEncryptionMinimal
=== RUN   TestAccECSTaskDefinition_EFSVolume_transitEncryption
=== PAUSE TestAccECSTaskDefinition_EFSVolume_transitEncryption
=== RUN   TestAccECSTaskDefinition_EFSVolume_transitEncryptionDisabled
=== PAUSE TestAccECSTaskDefinition_EFSVolume_transitEncryptionDisabled
=== RUN   TestAccECSTaskDefinition_EFSVolume_accessPoint
=== PAUSE TestAccECSTaskDefinition_EFSVolume_accessPoint
=== RUN   TestAccECSTaskDefinition_fsxWinFileSystem
=== PAUSE TestAccECSTaskDefinition_fsxWinFileSystem
=== RUN   TestAccECSTaskDefinition_DockerVolume_taskScoped
=== PAUSE TestAccECSTaskDefinition_DockerVolume_taskScoped
=== RUN   TestAccECSTaskDefinition_service
=== PAUSE TestAccECSTaskDefinition_service
=== RUN   TestAccECSTaskDefinition_taskRoleARN
=== PAUSE TestAccECSTaskDefinition_taskRoleARN
=== RUN   TestAccECSTaskDefinition_networkMode
=== PAUSE TestAccECSTaskDefinition_networkMode
=== RUN   TestAccECSTaskDefinition_ipcMode
=== PAUSE TestAccECSTaskDefinition_ipcMode
=== RUN   TestAccECSTaskDefinition_pidMode
=== PAUSE TestAccECSTaskDefinition_pidMode
=== RUN   TestAccECSTaskDefinition_constraint
=== PAUSE TestAccECSTaskDefinition_constraint
=== RUN   TestAccECSTaskDefinition_changeVolumesForcesNewResource
=== PAUSE TestAccECSTaskDefinition_changeVolumesForcesNewResource
=== RUN   TestAccECSTaskDefinition_arrays
=== PAUSE TestAccECSTaskDefinition_arrays
=== RUN   TestAccECSTaskDefinition_Fargate_basic
=== PAUSE TestAccECSTaskDefinition_Fargate_basic
=== RUN   TestAccECSTaskDefinition_Fargate_ephemeralStorage
=== PAUSE TestAccECSTaskDefinition_Fargate_ephemeralStorage
=== RUN   TestAccECSTaskDefinition_executionRole
=== PAUSE TestAccECSTaskDefinition_executionRole
=== RUN   TestAccECSTaskDefinition_disappears
=== PAUSE TestAccECSTaskDefinition_disappears
=== RUN   TestAccECSTaskDefinition_tags
=== PAUSE TestAccECSTaskDefinition_tags
=== RUN   TestAccECSTaskDefinition_proxy
=== PAUSE TestAccECSTaskDefinition_proxy
=== RUN   TestAccECSTaskDefinition_inferenceAccelerator
=== PAUSE TestAccECSTaskDefinition_inferenceAccelerator
=== RUN   TestAccECSTaskDefinition_invalidContainerDefinition
=== PAUSE TestAccECSTaskDefinition_invalidContainerDefinition
=== RUN   TestValidTaskDefinitionContainerDefinitions
=== PAUSE TestValidTaskDefinitionContainerDefinitions
=== RUN   TestAccECSTaskExecutionDataSource_basic
=== PAUSE TestAccECSTaskExecutionDataSource_basic
=== RUN   TestAccECSTaskExecutionDataSource_overrides
=== PAUSE TestAccECSTaskExecutionDataSource_overrides
=== RUN   TestAccECSTaskExecutionDataSource_tags
=== PAUSE TestAccECSTaskExecutionDataSource_tags
=== RUN   TestAccECSTaskSet_basic
=== PAUSE TestAccECSTaskSet_basic
=== RUN   TestAccECSTaskSet_withExternalId
=== PAUSE TestAccECSTaskSet_withExternalId
=== RUN   TestAccECSTaskSet_withScale
=== PAUSE TestAccECSTaskSet_withScale
=== RUN   TestAccECSTaskSet_disappears
=== PAUSE TestAccECSTaskSet_disappears
=== RUN   TestAccECSTaskSet_withCapacityProviderStrategy
=== PAUSE TestAccECSTaskSet_withCapacityProviderStrategy
=== RUN   TestAccECSTaskSet_withAlb
=== PAUSE TestAccECSTaskSet_withAlb
=== RUN   TestAccECSTaskSet_withLaunchTypeFargate
=== PAUSE TestAccECSTaskSet_withLaunchTypeFargate
=== RUN   TestAccECSTaskSet_withLaunchTypeFargateAndPlatformVersion
=== PAUSE TestAccECSTaskSet_withLaunchTypeFargateAndPlatformVersion
=== RUN   TestAccECSTaskSet_withServiceRegistries
=== PAUSE TestAccECSTaskSet_withServiceRegistries
=== RUN   TestAccECSTaskSet_tags
=== PAUSE TestAccECSTaskSet_tags
=== CONT  TestValidPlacementConstraint
=== CONT  TestAccECSTaskSet_tags
--- PASS: TestValidPlacementConstraint (0.00s)
=== CONT  TestAccECSService_ServiceRegistries_basic
=== CONT  TestAccECSService_replicaSchedulingStrategy
=== CONT  TestAccECSService_DaemonSchedulingStrategy_setDeploymentMinimum
=== CONT  TestAccECSService_DaemonSchedulingStrategy_basic
=== CONT  TestAccECSService_LaunchTypeEC2_network
=== CONT  TestAccECSService_LaunchTypeFargate_updateWaitForSteadyState
=== CONT  TestAccECSService_LaunchTypeFargate_waitForSteadyState
=== CONT  TestAccECSService_LaunchTypeFargate_platformVersion
=== CONT  TestAccECSService_ServiceRegistries_container
--- PASS: TestAccECSService_DaemonSchedulingStrategy_basic (34.42s)
=== CONT  TestAccECSService_LaunchTypeFargate_basic
--- PASS: TestAccECSService_DaemonSchedulingStrategy_setDeploymentMinimum (40.79s)
=== CONT  TestAccECSService_PlacementConstraints_emptyExpression
--- PASS: TestAccECSService_replicaSchedulingStrategy (71.22s)
=== CONT  TestAccECSService_PlacementConstraints_basic
--- PASS: TestAccECSTaskSet_tags (78.60s)
=== CONT  TestAccECSService_PlacementStrategy_missing
--- PASS: TestAccECSService_PlacementStrategy_missing (1.29s)
=== CONT  TestAccECSService_PlacementStrategy_basic
--- PASS: TestAccECSService_LaunchTypeEC2_network (104.07s)
=== CONT  TestAccECSTaskSet_withServiceRegistries
--- PASS: TestAccECSService_PlacementConstraints_emptyExpression (65.64s)
=== CONT  TestAccECSService_forceNewDeploymentTriggers
--- PASS: TestAccECSService_ServiceRegistries_basic (157.57s)
=== CONT  TestAccECSTaskSet_withLaunchTypeFargateAndPlatformVersion
--- PASS: TestAccECSService_ServiceRegistries_container (157.63s)
=== CONT  TestAccECSService_forceNewDeployment
--- PASS: TestAccECSService_PlacementStrategy_basic (80.22s)
=== CONT  TestAccECSTaskSet_withLaunchTypeFargate
--- PASS: TestAccECSService_PlacementConstraints_basic (89.73s)
=== CONT  TestAccECSService_multipleTargetGroups
--- PASS: TestAccECSService_LaunchTypeFargate_platformVersion (161.35s)
=== CONT  TestAccECSTaskSet_withAlb
--- PASS: TestAccECSService_LaunchTypeFargate_updateWaitForSteadyState (167.17s)
=== CONT  TestAccECSService_alb
--- PASS: TestAccECSService_LaunchTypeFargate_waitForSteadyState (167.75s)
=== CONT  TestAccECSTaskSet_withCapacityProviderStrategy
--- PASS: TestAccECSService_forceNewDeploymentTriggers (77.10s)
=== CONT  TestAccECSCluster_containerInsights
--- PASS: TestAccECSService_LaunchTypeFargate_basic (149.51s)
=== CONT  TestAccECSTaskSet_disappears
--- PASS: TestAccECSTaskSet_withLaunchTypeFargate (58.99s)
=== CONT  TestAccECSCluster_serviceConnectDefaults
--- PASS: TestAccECSTaskSet_disappears (44.27s)
=== CONT  TestAccECSTaskSet_withScale
--- PASS: TestAccECSTaskSet_withServiceRegistries (125.20s)
=== CONT  TestAccECSCluster_tags
--- PASS: TestAccECSTaskSet_withLaunchTypeFargateAndPlatformVersion (82.09s)
=== CONT  TestAccECSTaskSet_withExternalId
--- PASS: TestAccECSService_forceNewDeployment (89.11s)
=== CONT  TestAccECSCluster_disappears
--- PASS: TestAccECSCluster_containerInsights (70.64s)
=== CONT  TestAccECSTaskSet_basic
--- PASS: TestAccECSTaskSet_withCapacityProviderStrategy (94.83s)
=== CONT  TestAccECSCluster_basic
--- PASS: TestAccECSCluster_disappears (27.65s)
=== CONT  TestAccECSTaskExecutionDataSource_tags
--- PASS: TestAccECSCluster_tags (53.27s)
=== CONT  TestAccECSClusterDataSource_tags
--- PASS: TestAccECSTaskSet_withExternalId (52.70s)
=== CONT  TestAccECSTaskExecutionDataSource_overrides
--- PASS: TestAccECSCluster_basic (29.97s)
=== CONT  TestAccECSClusterDataSource_ecsClusterContainerInsights
--- PASS: TestAccECSTaskSet_withScale (66.24s)
=== CONT  TestAccECSTaskExecutionDataSource_basic
--- PASS: TestAccECSClusterDataSource_tags (26.17s)
=== CONT  TestAccECSClusterDataSource_ecsCluster
--- PASS: TestAccECSTaskSet_basic (56.22s)
=== CONT  TestAccECSClusterCapacityProviders_Update_defaultStrategy
--- PASS: TestAccECSClusterDataSource_ecsClusterContainerInsights (26.70s)
=== CONT  TestValidTaskDefinitionContainerDefinitions
--- PASS: TestValidTaskDefinitionContainerDefinitions (0.00s)
=== CONT  TestAccECSClusterCapacityProviders_Update_capacityProviders
--- PASS: TestAccECSClusterDataSource_ecsCluster (26.46s)
=== CONT  TestAccECSTaskDefinition_invalidContainerDefinition
--- PASS: TestAccECSTaskDefinition_invalidContainerDefinition (1.16s)
=== CONT  TestAccECSClusterCapacityProviders_destroy
--- PASS: TestAccECSCluster_serviceConnectDefaults (144.15s)
=== CONT  TestAccECSTaskDefinition_inferenceAccelerator
=== NAME  TestAccECSClusterCapacityProviders_destroy
    cluster_capacity_providers_test.go:120: Step 1/2 error: Check failed: Check 2/2 error: RegisteredContainerInstancesCount = 0, want 2
--- PASS: TestAccECSTaskDefinition_inferenceAccelerator (18.22s)
=== CONT  TestAccECSClusterCapacityProviders_defaults
--- PASS: TestAccECSTaskSet_withAlb (225.91s)
=== CONT  TestAccECSClusterCapacityProviders_disappears
--- PASS: TestAccECSService_multipleTargetGroups (269.67s)
=== CONT  TestAccECSClusterCapacityProviders_basic
--- PASS: TestAccECSClusterCapacityProviders_defaults (51.47s)
=== CONT  TestAccECSCapacityProvider_tags
--- PASS: TestAccECSService_alb (266.13s)
=== CONT  TestAccECSCapacityProvider_managedScalingPartial
--- PASS: TestAccECSClusterCapacityProviders_Update_defaultStrategy (128.49s)
=== CONT  TestAccECSCapacityProvider_managedScaling
--- PASS: TestAccECSClusterCapacityProviders_disappears (58.87s)
=== CONT  TestAccECSCapacityProvider_disappears
=== CONT  TestAccECSCapacityProvider_basic
--- PASS: TestAccECSClusterCapacityProviders_Update_capacityProviders (128.30s)
--- PASS: TestAccECSCapacityProvider_managedScalingPartial (47.53s)
=== CONT  TestAccECSService_healthCheckGracePeriodSeconds
--- PASS: TestAccECSCapacityProvider_disappears (35.95s)
=== CONT  TestAccECSAccountSettingDefault_containerInsights
=== CONT  TestAccECSAccountSettingDefault_vpcTrunking
--- PASS: TestAccECSClusterCapacityProviders_basic (52.58s)
--- FAIL: TestAccECSClusterCapacityProviders_destroy (150.93s)
=== CONT  TestAccECSService_clusterName
--- PASS: TestAccECSCapacityProvider_basic (47.98s)
=== CONT  TestAccECSAccountSettingDefault_taskLongARNFormat
--- PASS: TestAccECSAccountSettingDefault_containerInsights (19.04s)
=== CONT  TestAccECSAccountSettingDefault_serviceLongARNFormat
--- PASS: TestAccECSAccountSettingDefault_vpcTrunking (18.88s)
=== CONT  TestAccECSService_loadBalancerChanges
--- PASS: TestAccECSCapacityProvider_managedScaling (71.71s)
=== CONT  TestAccECSAccountSettingDefault_containerInstanceLongARNFormat
--- PASS: TestAccECSCapacityProvider_tags (80.89s)
=== CONT  TestAccECSCluster_configuration
--- PASS: TestAccECSAccountSettingDefault_taskLongARNFormat (18.49s)
=== CONT  TestAccECSService_deploymentCircuitBreaker
--- PASS: TestAccECSAccountSettingDefault_serviceLongARNFormat (18.59s)
=== CONT  TestValidPlacementStrategy
--- PASS: TestValidPlacementStrategy (0.00s)
=== CONT  TestAccECSService_familyAndRevision
--- PASS: TestAccECSAccountSettingDefault_containerInstanceLongARNFormat (18.45s)
=== CONT  TestAccECSService_DeploymentValues_minZeroMaxOneHundred
--- PASS: TestAccECSTaskExecutionDataSource_tags (285.07s)
=== CONT  TestAccECSTaskDefinition_DockerVolume_minimal
--- PASS: TestAccECSService_clusterName (77.11s)
=== CONT  TestAccECSService_CapacityProviderStrategy_update
--- PASS: TestAccECSCluster_configuration (56.72s)
=== CONT  TestAccECSService_DeploymentValues_basic
--- PASS: TestAccECSTaskDefinition_DockerVolume_minimal (18.55s)
=== CONT  TestAccECSTaskDefinition_service
--- PASS: TestAccECSService_deploymentCircuitBreaker (65.66s)
=== CONT  TestAccECSService_CapacityProviderStrategy_forceNewDeployment
--- PASS: TestAccECSTaskExecutionDataSource_overrides (290.26s)
=== CONT  TestAccECSTaskDefinition_DockerVolume_basic
--- PASS: TestAccECSTaskExecutionDataSource_basic (289.90s)
=== CONT  TestAccECSTaskDefinition_fsxWinFileSystem
--- PASS: TestAccECSService_familyAndRevision (75.70s)
=== CONT  TestAccECSService_CapacityProviderStrategy_basic
--- PASS: TestAccECSService_DeploymentValues_minZeroMaxOneHundred (66.39s)
=== CONT  TestAccECSTaskDefinition_scratchVolume
--- PASS: TestAccECSTaskDefinition_DockerVolume_basic (18.94s)
=== CONT  TestAccECSTaskDefinition_EFSVolume_accessPoint
--- PASS: TestAccECSTaskDefinition_scratchVolume (19.09s)
=== CONT  TestAccECSService_PlacementStrategy_unnormalized
--- PASS: TestAccECSTaskDefinition_EFSVolume_accessPoint (32.32s)
=== CONT  TestAccECSTaskDefinition_basic
--- PASS: TestAccECSService_DeploymentValues_basic (65.95s)
=== CONT  TestAccECSTaskDefinition_EFSVolume_transitEncryptionDisabled
--- PASS: TestAccECSTaskDefinition_service (83.56s)
=== CONT  Test_StripRevision
=== RUN   Test_StripRevision/empty
=== PAUSE Test_StripRevision/empty
=== RUN   Test_StripRevision/invalid_arn
=== PAUSE Test_StripRevision/invalid_arn
=== RUN   Test_StripRevision/with_revision
=== PAUSE Test_StripRevision/with_revision
=== RUN   Test_StripRevision/no_revision
=== PAUSE Test_StripRevision/no_revision
=== CONT  TestAccECSTaskDefinition_EFSVolume_transitEncryption
--- PASS: TestAccECSTaskDefinition_basic (29.42s)
=== CONT  TestContainerDefinitionsAreEquivalent_missingEnvironmentName
--- PASS: TestContainerDefinitionsAreEquivalent_missingEnvironmentName (0.00s)
=== CONT  TestAccECSTaskDefinition_EFSVolume_transitEncryptionMinimal
--- PASS: TestAccECSTaskDefinition_EFSVolume_transitEncryptionDisabled (29.27s)
=== CONT  TestAccECSTaskDefinition_EFSVolume_basic
--- PASS: TestAccECSService_PlacementStrategy_unnormalized (66.09s)
=== CONT  TestContainerDefinitionsAreEquivalent_negative
--- PASS: TestContainerDefinitionsAreEquivalent_negative (0.00s)
=== CONT  TestAccECSTaskDefinition_EFSVolume_minimal
--- PASS: TestAccECSTaskDefinition_EFSVolume_transitEncryption (29.68s)
=== CONT  TestContainerDefinitionsAreEquivalent_arrays
--- PASS: TestContainerDefinitionsAreEquivalent_arrays (0.00s)
=== CONT  TestAccECSTaskDefinition_Fargate_runtimePlatformWithoutArch
--- PASS: TestAccECSTaskDefinition_EFSVolume_transitEncryptionMinimal (28.95s)
=== CONT  TestAccECSTaskDefinition_Fargate_runtimePlatform
--- PASS: TestAccECSTaskDefinition_EFSVolume_basic (29.03s)
=== CONT  TestContainerDefinitionsAreEquivalent_portMappingsIgnoreHostPort
--- PASS: TestContainerDefinitionsAreEquivalent_portMappingsIgnoreHostPort (0.00s)
=== CONT  TestAccECSTaskDefinition_runtimePlatform
--- PASS: TestAccECSService_CapacityProviderStrategy_forceNewDeployment (116.34s)
=== CONT  TestContainerDefinitionsAreEquivalent_portMappings
--- PASS: TestContainerDefinitionsAreEquivalent_portMappings (0.00s)
=== CONT  TestAccECSTaskDefinition_Fargate_basic
--- PASS: TestAccECSTaskDefinition_EFSVolume_minimal (30.98s)
=== CONT  TestContainerDefinitionsAreEquivalent_basic
--- PASS: TestContainerDefinitionsAreEquivalent_basic (0.00s)
=== CONT  TestAccECSTaskDefinition_proxy
--- PASS: TestAccECSTaskDefinition_Fargate_runtimePlatformWithoutArch (20.71s)
=== CONT  TestAccECSService_alarmsUpdate
--- PASS: TestAccECSTaskDefinition_Fargate_runtimePlatform (20.73s)
=== CONT  TestAccECSTaskDefinition_tags
--- PASS: TestAccECSTaskDefinition_runtimePlatform (21.07s)
=== CONT  TestAccECSTaskDefinition_taskRoleARN
--- PASS: TestAccECSTaskDefinition_Fargate_basic (27.21s)
=== CONT  TestAccECSTaskDefinition_disappears
--- PASS: TestAccECSService_CapacityProviderStrategy_basic (134.93s)
=== CONT  TestAccECSTaskDefinition_DockerVolume_taskScoped
--- PASS: TestAccECSTaskDefinition_taskRoleARN (22.38s)
=== CONT  TestAccECSTaskDefinition_Fargate_ephemeralStorage
--- PASS: TestAccECSTaskDefinition_proxy (31.55s)
=== CONT  TestAccECSTaskDefinition_arrays
--- PASS: TestAccECSTaskDefinition_DockerVolume_taskScoped (17.37s)
=== CONT  TestAccECSService_alarmsAdd
--- PASS: TestAccECSTaskDefinition_disappears (29.65s)
=== CONT  TestAccECSTaskDefinition_changeVolumesForcesNewResource
--- PASS: TestAccECSTaskDefinition_Fargate_ephemeralStorage (20.65s)
=== CONT  TestAccECSService_DeploymentControllerType_external
--- PASS: TestAccECSService_loadBalancerChanges (259.53s)
=== CONT  Test_GetClustereNameFromARN
=== RUN   Test_GetClustereNameFromARN/empty
=== PAUSE Test_GetClustereNameFromARN/empty
=== RUN   Test_GetClustereNameFromARN/cluster
=== PAUSE Test_GetClustereNameFromARN/cluster
=== CONT  TestAccECSTaskDefinition_constraint
--- PASS: TestAccECSTaskDefinition_arrays (20.71s)
=== CONT  TestAccECSService_DeploymentControllerType_codeDeployUpdateDesiredCountAndHealthCheckGracePeriod
--- PASS: TestAccECSTaskDefinition_tags (57.06s)
=== CONT  TestAccECSTaskDefinition_pidMode
--- PASS: TestAccECSTaskDefinition_constraint (22.96s)
=== CONT  TestAccECSService_DeploymentControllerType_codeDeploy
--- PASS: TestAccECSTaskDefinition_changeVolumesForcesNewResource (32.26s)
=== CONT  TestAccECSTaskDefinition_ipcMode
--- PASS: TestAccECSService_healthCheckGracePeriodSeconds (309.41s)
=== CONT  TestAccECSService_iamRole
--- PASS: TestAccECSTaskDefinition_pidMode (21.21s)
=== CONT  TestAccECSTaskDefinition_networkMode
--- PASS: TestAccECSService_alarmsUpdate (82.64s)
=== CONT  TestAccECSService_Tags_managed
--- PASS: TestAccECSService_DeploymentControllerType_external (40.40s)
=== CONT  TestAccECSService_ServiceConnect_full
--- PASS: TestAccECSTaskDefinition_ipcMode (21.00s)
=== CONT  TestAccECSTag_value
--- PASS: TestAccECSService_alarmsAdd (59.29s)
=== CONT  TestAccECSService_Tags_basic
--- PASS: TestAccECSTaskDefinition_networkMode (20.19s)
=== CONT  TestAccECSTag_ResourceARN_batchComputeEnvironment
--- PASS: TestAccECSTag_ResourceARN_batchComputeEnvironment (31.53s)
=== CONT  TestAccECSTag_basic
--- PASS: TestAccECSService_iamRole (53.29s)
=== CONT  TestAccECSTaskDefinition_executionRole
--- PASS: TestAccECSTag_value (43.08s)
=== CONT  TestAccECSTag_disappears
--- PASS: TestAccECSService_Tags_managed (65.90s)
=== CONT  TestAccECSService_ServiceConnect_remove
--- PASS: TestAccECSTaskDefinition_executionRole (20.95s)
=== CONT  TestAccECSService_basic
--- PASS: TestAccECSTag_basic (31.26s)
=== CONT  TestAccECSService_ServiceConnect_ingressPortOverride
--- PASS: TestAccECSTag_disappears (27.51s)
=== CONT  TestAccECSService_ServiceRegistries_removal
--- PASS: TestAccECSService_Tags_basic (92.96s)
=== CONT  TestAccECSService_ServiceConnect_basic
--- PASS: TestAccECSService_basic (78.45s)
=== CONT  TestAccECSService_Tags_propagate
--- PASS: TestAccECSService_ServiceConnect_full (174.26s)
=== CONT  TestAccECSService_ServiceRegistries_changes
--- PASS: TestAccECSService_ServiceRegistries_removal (130.42s)
=== CONT  TestAccECSServiceDataSource_basic
--- PASS: TestAccECSService_ServiceConnect_remove (169.29s)
=== CONT  Test_GetRoleNameFromARN
=== RUN   Test_GetRoleNameFromARN/empty
=== PAUSE Test_GetRoleNameFromARN/empty
=== RUN   Test_GetRoleNameFromARN/role
=== PAUSE Test_GetRoleNameFromARN/role
=== RUN   Test_GetRoleNameFromARN/role_with_path
=== PAUSE Test_GetRoleNameFromARN/role_with_path
=== RUN   Test_GetRoleNameFromARN/role_with_complex_path
=== PAUSE Test_GetRoleNameFromARN/role_with_complex_path
=== CONT  TestAccECSContainerDefinitionDataSource_ecsContainerDefinition
--- PASS: TestAccECSService_Tags_propagate (90.59s)
=== CONT  TestAccECSService_basicImport
--- PASS: TestAccECSService_ServiceConnect_ingressPortOverride (175.77s)
=== CONT  TestAccECSTaskDefinitionDataSource_ecsTaskDefinition
--- PASS: TestAccECSService_DeploymentControllerType_codeDeploy (269.52s)
=== CONT  TestAccECSService_executeCommand
--- PASS: TestAccECSService_ServiceConnect_basic (156.49s)
=== CONT  TestAccECSService_disappears
--- PASS: TestAccECSTaskDefinitionDataSource_ecsTaskDefinition (17.75s)
=== CONT  Test_StripRevision/empty
=== CONT  Test_StripRevision/with_revision
=== CONT  Test_StripRevision/no_revision
=== CONT  Test_StripRevision/invalid_arn
--- PASS: Test_StripRevision (0.00s)
    --- PASS: Test_StripRevision/empty (0.00s)
    --- PASS: Test_StripRevision/with_revision (0.00s)
    --- PASS: Test_StripRevision/no_revision (0.00s)
    --- PASS: Test_StripRevision/invalid_arn (0.00s)
=== CONT  Test_GetClustereNameFromARN/empty
=== CONT  Test_GetClustereNameFromARN/cluster
--- PASS: Test_GetClustereNameFromARN (0.00s)
    --- PASS: Test_GetClustereNameFromARN/empty (0.00s)
    --- PASS: Test_GetClustereNameFromARN/cluster (0.00s)
=== CONT  Test_GetRoleNameFromARN/empty
=== CONT  Test_GetRoleNameFromARN/role_with_path
=== CONT  Test_GetRoleNameFromARN/role_with_complex_path
=== CONT  Test_GetRoleNameFromARN/role
--- PASS: Test_GetRoleNameFromARN (0.00s)
    --- PASS: Test_GetRoleNameFromARN/empty (0.00s)
    --- PASS: Test_GetRoleNameFromARN/role_with_path (0.00s)
    --- PASS: Test_GetRoleNameFromARN/role_with_complex_path (0.00s)
    --- PASS: Test_GetRoleNameFromARN/role (0.00s)
--- PASS: TestAccECSServiceDataSource_basic (65.96s)
--- PASS: TestAccECSContainerDefinitionDataSource_ecsContainerDefinition (76.67s)
--- PASS: TestAccECSService_executeCommand (89.01s)
--- PASS: TestAccECSService_DeploymentControllerType_codeDeployUpdateDesiredCountAndHealthCheckGracePeriod (471.12s)
--- PASS: TestAccECSService_ServiceRegistries_changes (261.73s)
--- PASS: TestAccECSService_basicImport (202.11s)
--- PASS: TestAccECSService_disappears (186.19s)
--- PASS: TestAccECSService_CapacityProviderStrategy_update (792.75s)
=== NAME  TestAccECSTaskDefinition_fsxWinFileSystem
    task_definition_test.go:557: Step 1/2 error: Error running apply: exit status 1
        
        Error: waiting for FSx for Windows File Server File System (fs-02c2fde9f0e68984d) create: unexpected state 'FAILED', wanted target 'AVAILABLE'. last error: File system creation failed. Amazon FSx is unable to communicate with your Microsoft Active Directory domain controllers. Please delete your file system and create a new one once you’ve allowed network traffic between Amazon FSx and your domain controllers as recommended in the Amazon FSx user guide: https://docs.aws.amazon.com/fsx/latest/WindowsGuide/aws-ad-integration-fsxW.html
        
          with aws_fsx_windows_file_system.test,
          on terraform_plugin_test.tf line 54, in resource "aws_fsx_windows_file_system" "test":
          54: resource "aws_fsx_windows_file_system" "test" {
        
--- FAIL: TestAccECSTaskDefinition_fsxWinFileSystem (3288.61s)
FAIL
FAIL	github.com/hashicorp/terraform-provider-aws/internal/service/ecs	3879.534s
FAIL

Reply:
Test runs of full and basic only:

==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./internal/service/ecs/... -v -count 1 -parallel 10 -run='TestAccECSService_ServiceConnect_full'  -timeout 360m
=== RUN   TestAccECSService_ServiceConnect_full
=== PAUSE TestAccECSService_ServiceConnect_full
=== CONT  TestAccECSService_ServiceConnect_full
--- PASS: TestAccECSService_ServiceConnect_full (177.84s)
PASS
ok      github.com/hashicorp/terraform-provider-aws/internal/service/ecs        183.620s


TF_ACC=1 go test ./internal/service/ecs/... -v -count 1 -parallel 10 -run='TestAccECSService_ServiceConnect_basic'  -timeout 360m
=== RUN   TestAccECSService_ServiceConnect_basic
=== PAUSE TestAccECSService_ServiceConnect_basic
=== CONT  TestAccECSService_ServiceConnect_basic
--- PASS: TestAccECSService_ServiceConnect_basic (168.87s)
PASS
ok      github.com/hashicorp/terraform-provider-aws/internal/service/ecs        174.638s